pageserver/src/http/routes.rs

```diff
@@ -519,14 +519,16 @@ async fn tenant_size_handler(request: Request<Body>) -> Result<Response<Body>, A
         ///
         /// Will be none if `?inputs_only=true` was given.
         size: Option<u64>,
+        segment_sizes: Option<Vec<tenant_size_model::SegmentSizeResult>>,
```

Comment:
Should this be noted above in the type comment next to '"unstable" `inputs` field' or put into swagger desc?

Reply:
Unstable it is, no swagger desc yet, resolving.